src/app/view/dialogs/note-dialog/note-filter-tab/note-filter-tab.component.ts

```diff
@@ -64,7 +71,9 @@ export class NoteFilterTabComponent implements OnInit, OnDestroy {
     this.noteLabels = this.noteLabels.filter(
       (labels) => labels !== valueDelete,
     );
-    this.noteService.setLabels(this.note.getId(), this.noteLabels);
+    this.isLabelBeingEdited = true;
```

Comment:
Hm, do we really need these booleans? Can't we pass a flag via function arguments instead?

How does `onLabelsFocusout` end up being called by `checkAndSetLabels`?